src/main/java/org/codehaus/mojo/gwt/AbstractGwtMojo.java

```diff
@@ -293,6 +321,17 @@ protected boolean isGenerator() {
         return files;
     }
 
+    private List<Exclusion> getProjectPluginExclusions() {
+      final List<Exclusion> exclusions = new ArrayList<Exclusion>();
+      final Plugin plugin = project.getPlugin(Plugin.constructKey(GWT_PLUGIN_GROUP_ID, GWT_PLUGIN_ARTIFACT_ID));
+      for ( final Dependency dep : plugin.getDependencies() )
+      {
+        exclusions.addAll( dep.getExclusions() );
```

Comment:
Sorry, I do not understand the intention of this. Looks like you merge all exclusion from all dependencies, and later on you filter all dependencies by this aggregate exclusions, is this correct? Will be helpful a test case, this also helps to understand the intention 😉. OTOH, this looks a bit hacky, you can always create a custom version with this and deploy to your internal repo. This plugin is totally stable and feature freeze, so maintainability is not a problem.